src/reuser/agent/ReuserAgent.php

```diff
@@ -75,15 +75,41 @@ function processUploadId($uploadId)
       {
         continue;
       }
-      if (empty($reuseMode)) {
-        $this->processUploadReuse($itemTreeBounds, $itemTreeBoundsReused, $reusedGroupId);
-      } else {
+      if($reuseMode&(1<<1)){
```

Comment:
use `ReuserAgentPlugin::REUSE_ENHANCED` instead of `1<<1` to increase readability